src/uwtools/config.py

```diff
@@ -116,10 +154,11 @@ def __init__(self, config_path):
 
         self.update(self._load())
 
-    def _load(self):
+    def _load(self, config_path=None):
         ''' Load the user-provided Fortran namelist path into a dict
         object. '''
-        with open(self.config_path, 'r', encoding="utf-8") as file_name:
+        config_path = config_path or self.config_path
+        with open(config_path, 'r', encoding="utf-8") as file_name:
```

Comment:
@kevenmblackman I am the worst about remembering to add try/except/finally blocks. Here, I was thinking that maybe it is not explicitly needed, though.

I haven't typically seen try/finally blocks used alongside context managers (with-statement). It's my understanding that they are essentially doing the same task. The `__enter__` and `__exit__` methods on the `file` object returned by `open()` work with the `with` so that they are called before and after the code inside the `with` statement.

I'm curious if you think we'd need both? Or if one way is more robust than the other? I've typically gone with the with-statement because it reduces syntax and eliminates the need for me to remember to close a file (the lazy-coder reasons).

Reply:
@kevenmblackman On PR #116, @nbharwani11 found that when replacing the with-statement with a try/finally, the linter suggested that a with-statement is more in line with the PEP-8 standard. 

We can always have a project-based exception to the standards, but I think we'd need to have a better understanding of why we might not want a with-statement before we make that change.

Reply:
Confirming what Christina mentioned, that the linter suggested a with-statement was more in line with the PEP-8 standards. Did we ever get confirmation on why we might want to use a try/finally over a with-statement? 

Otherwise naming conventions seemed to follow PEP-8 standards and the code looks good to me!

Reply:
I approved it, just write a follow on at this point as there are a few other methods in here doing that as well.  This will all be easier when I scan it in SonarQube later today, so we can automate most of this.